### PR TITLE
Fix docstrings warnings.

### PR DESCRIPTION
**Pull request by nadiaya**
Fix docstrings warnings.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/amazon/knn.py [hunk 1]
```diff
@@ -108,6 +109,7 @@ def create_model(self, vpc_config_override=VPC_CONFIG_DEFAULT):
                 * 'Subnets' (list[str]): List of subnet ids.
                 * 'SecurityGroupIds' (list[str]): List of security group ids.
         """
+
```
Comment (laurenyu) [inline src/sagemaker/amazon/knn.py]: really? this newline is required? 😕

File: src/sagemaker/sparkml/model.py [hunk 2]
```diff
@@ -57,6 +58,7 @@ class SparkMLModel(Model):
 
     def __init__(self, model_data, role=None, spark_version=2.2, sagemaker_session=None, **kwargs):
         """Initialize a SparkMLModel..
```
Comment (laurenyu) [inline src/sagemaker/sparkml/model.py]: remove the second period

File: src/sagemaker/tensorflow/estimator.py [hunk 3]
```diff
@@ -177,6 +177,7 @@ def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=N
                  framework_version=None, model_dir=None, requirements_file='', image_name=None,
                  script_mode=False, distributions=None, **kwargs):
         """Initialize an ``TensorFlow`` estimator.
```
Comment (laurenyu) [inline src/sagemaker/tensorflow/estimator.py]: s/an/a

File: src/sagemaker/tensorflow/model.py [hunk 4]
```diff
@@ -21,9 +21,9 @@
 
 
 class TensorFlowPredictor(RealTimePredictor):
-    """A ``RealTimePredictor`` for inference against TensorFlow ``Endpoint``s.
+    """A ``RealTimePredictor`` for inference against TensorFlow ``Endpoint``.
```
Comment (laurenyu) [inline src/sagemaker/tensorflow/model.py]: I'd change this to "a TensorFlow endpoint"

File: src/sagemaker/amazon/knn.py (hunk 1 above)
Comment (nadiaya) [reply src/sagemaker/amazon/knn.py]: not sure, but we had it in other places, so for consistency :)

Comment (laurenyu) [reply src/sagemaker/amazon/knn.py]: I think in most places we try not to include that newline: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

**Pull request merged**